crates/swc_ecma_transforms_optimization/src/simplify/expr/tests.rs

```diff
@@ -1544,3 +1544,8 @@ fn test_es6_features() {
         "function foo() {return `${false}`}",
     );
 }
+
+#[test]
+fn test_export_default_paren_expr() {
+    fold_same("import fn from './b'; export default (function fn () {});");
```

Comment:
Another case should be added

```JavaScript
export default ((foo));
```

and expected output:

```JavaScript
export default foo;
```

Reply:
@magic-akari i resolved it, please review it again.